spinalcordtoolbox/reports/assets/index.html

```diff
@@ -324,17 +324,18 @@
     function download_yaml(marker) {
         let res = {};
         let seg_cmd_to_seg_type = {
-            "sct_analyze_lesion": "FILES_GMSEG",
+            "sct_analyze_lesion": "FILES_LESION",
             "sct_analyze_texture": "FILES_SEG",
             "sct_create_mask": "FILES_SEG",
             "sct_crop_image": "FILES_SEG",
             "sct_deepseg": "FILES_SEG",
             "sct_deepseg_gm": "FILES_GMSEG",
-            "sct_deepseg_lesion": "FILES_GMSEG",
+            "sct_deepseg_lesion": "FILES_LESION",
             "sct_deepseg_sc": "FILES_SEG",
             "sct_detect_pmj": "FILES_PMJ",
             "sct_dmri_moco": "FILES_TIMESERIES",
             "sct_fmri_moco": "FILES_TIMESERIES",
+            "sct_image_stitch": "FILES_REG",
```

Comment:
Note: I chose to reuse the existing `FILES_REG` for `sct_image -stitch` because I figured that stitching images together can sort of be seen as "registering multiple images together in the same space"?

If this feels wrong, though, then maybe we can create a new key type. Not sure? :thinking:

Reply:
That makes sense to me!